sdk/eventgrid/eventgrid-namespaces/src/eventGridNamespacesPublishBinaryMode.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  isUnexpected,
+  EventGridContext as Client,
+  PublishCloudEvent200Response,
+  PublishCloudEventDefaultResponse,
+} from "./cadl-generated/rest/index";
+import { StreamableMethod, operationOptionsToRequestParameters } from "@azure-rest/core-client";
+import { PublishCloudEventOptions } from "./cadl-generated/models/options";
+import { RawHttpHeadersInput } from "@azure/core-rest-pipeline";
+import { Buffer } from "buffer";
+import { CloudEvent } from "./cadl-generated";
+
+export async function publishCloudEventBinaryMode(
```

Comment:
`publishCloudEventInBinaryMode`?

Reply:
It is not exposed outside of SDK. But, Yes. The new name is more meaningful. I have changed it (in the next commit)